src/unix/linux_like/linux/mod.rs

```diff
@@ -2620,6 +2620,13 @@ f! {
         }
     }
 
+    pub fn CPU_ALLOC_SIZE(count: ::size_t) -> ::c_int {
```

Comment:
I think it should be?

```suggestion
    pub fn CPU_ALLOC_SIZE(count: ::c_int) -> ::size_t {
```